activejob-traceable.gemspec

```diff
@@ -8,7 +10,7 @@ Gem::Specification.new do |spec|
   spec.authors       = ['Dmytro Zakharov']
   spec.email         = ['dmytro@qonto.eu']
```

Comment:
can you update the email to be `backend@qonto.eu`, please?

Reply:
Sure!